pixeltable/functions/timestamp.py

```diff
@@ -232,7 +232,7 @@ def _(
         sql.cast(day, sql.Integer),
         sql.cast(hour, sql.Integer),
         sql.cast(minute, sql.Integer),
-        sql.cast(second + microsecond / 1000000.0, sql.Double))
+        sql.cast(second + microsecond / 1000000.0, sql.Float))
```

Comment:
What does this do?

Reply:
SQL generated before: CAST(EXTRACT(second FROM tbl_86ac1840fd56429798fffecb6dc1204a.col_0) + 0 / CAST(1000000.0 AS FLOAT) AS **_DOUBLE_**)) 
SQL generated now: CAST(EXTRACT(second FROM tbl_86ac1840fd56429798fffecb6dc1204a.col_0) + 0 / CAST(1000000.0 AS FLOAT) AS **_FLOAT_**))